SignalServiceKit/src/Contacts/Threads/TSGroupThread.m

```diff
@@ -162,12 +176,23 @@ - (NSString *)name
 
 - (void)updateAvatarWithAttachmentStream:(TSAttachmentStream *)attachmentStream
```

Comment:
I'm ambivalent - I definitely am sympathetic to your motive, but I'm not sure its practical to remove all new transactions from our models since this is so pervasive in our codebase.  And it would involve a lot more verbosity.  

And if it's alright with you, I'd rather not make those changes in this PR which is already huge. Let's discuss.